### PR TITLE
Add Hacktoberfest 2024 post

### PR DESCRIPTION
**Pull request by TraceyC77**
Add post for Hacktoberfest 2024

### CODE REVIEW

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+categories:
+- news
+date: 2024-09-30T20:29:55-05:00
+featuredimage: "/horizontal_black.png"
+url: "/2024/09/30/solus-hacktoberfest-2024"
+---
+
+Hey folks! It's spooky season, so you know what that means. It's time for [Hacktoberfest 2024](https://hacktoberfest.com/)! This is the annual event that "encourages people to contribute to open source throughout October", in exchange for rewards.
+
+This year we have two main goals. First, as always, we want to support anyone looking to get involved in Open Source contributions, or helping Solus specifically. We would love to add people to our packaging team.
```
Comment (Staudey) [inline content/blog/2024/09/solus-hacktoberfest-2024/index.md:13]:
```suggestion
This year we have two main goals. First, as always, we want to support anyone looking to get involved in open source contributions, or helping Solus specifically. We would love to add people to our packaging team.
```

It's written as "open source" with small first letters in all other places in the post. Not sure which format is "better", but it should be uniform.

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+## OK so how do I get involved? And who can help me if I need it?
+
+- [Register](https://hacktoberfest.com/auth/) on the Hacktoberfest website
+- Read up on [what it is and how to participate](https://hacktoberfest.com/about/)
+- Check out our [docs on packaging](https://help.getsol.us/docs/packaging)
+- Join our [Matrix space](https://matrix.to/#/#solus:matrix.org), we use the Solus Packaging room to discuss packaging work.
+- Find something(s) to work on
+  - For development / bug triage: look in the [Solus Packages Repo](https://github.com/getsolus/packages/issues):
+  - We also need help with some issues in [our Help Center repo](https://github.com/getsolus/help-center-docs/issue)
```
Comment (Staudey) [inline content/blog/2024/09/solus-hacktoberfest-2024/index.md:37]:
```suggestion
  - We also need help with some issues in [our Help Center repository](https://github.com/getsolus/help-center-docs/issue)
```

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+Anything labeled "Hacktoberfest", "Good First Issue", "Package Request" or "Help Wanted" is a good candidate for new folks (but don't feel limited to them)
+- Do work and create pull requests in the Solus repos
```
Comment (Staudey) [inline content/blog/2024/09/solus-hacktoberfest-2024/index.md:39]:
```suggestion
- Do work and create pull requests in the Solus repositories
```

**Review by Staudey — COMMENTED** [review]
Just some minor things

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+date: 2024-09-30T20:29:55-05:00
+featuredimage: "/horizontal_black.png"
+url: "/2024/09/30/solus-hacktoberfest-2024"
+---
+
+Hey folks! It's spooky season, so you know what that means. It's time for [Hacktoberfest 2024](https://hacktoberfest.com/)! This is the annual event that "encourages people to contribute to open source throughout October", in exchange for rewards.
+
+This year we have two main goals. First, as always, we want to support anyone looking to get involved in Open Source contributions, or helping Solus specifically. We would love to add people to our packaging team.
+
+Second, there's been a big push recently to clean up and update our package repository, and we can use your help. Much of this work currently involves adding home pages to packages, making sure they build, fixing them if the don't, and adding monitoring information. Related, we would like to field test and improve our "autobuild" tool.
```
[the remark this answers is not in the document]
Comment (TraceyC77) [reply content/blog/2024/09/solus-hacktoberfest-2024/index.md:15]: The Style Guide says to use Mirriam Websters dictionary, which has "home page". Agree with the grammatical correction to "they", fixing that.

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md (hunk 1 above)
Comment (TraceyC77) [reply content/blog/2024/09/solus-hacktoberfest-2024/index.md:13]: The Microsoft style guide uses lower case, which seems more correct to me anyway. Good catch.

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+Developers aren't the only contributors in the open source community. We also have people who write documentation, people who submit and triage bugs, and people who do communications and outreach. Your submission to a project does not have to be code.
+
+## OK so how do I get involved? And who can help me if I need it?
+
+- [Register](https://hacktoberfest.com/auth/) on the Hacktoberfest website
+- Read up on [what it is and how to participate](https://hacktoberfest.com/about/)
+- Check out our [docs on packaging](https://help.getsol.us/docs/packaging)
+- Join our [Matrix space](https://matrix.to/#/#solus:matrix.org), we use the Solus Packaging room to discuss packaging work.
+- Find something(s) to work on
+  - For development / bug triage: look in the [Solus Packages Repo](https://github.com/getsolus/packages/issues):
```
[the remark this answers is not in the document]
Comment (TraceyC77) [reply content/blog/2024/09/solus-hacktoberfest-2024/index.md:36]: You're right, fixing.

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md (hunk 2 above)
Comment (TraceyC77) [reply content/blog/2024/09/solus-hacktoberfest-2024/index.md:37]: Fixing

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md (hunk 3 above)
Comment (TraceyC77) [reply content/blog/2024/09/solus-hacktoberfest-2024/index.md:39]: Fixing

File: content/blog/2024/09/solus-hacktoberfest-2024/index.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Staudey) [reply content/blog/2024/09/solus-hacktoberfest-2024/index.md:15]: Indeed it has. Good to know.

**Pull request merged**